### DISCUSSION TITLE
Hello I need help setting up auto mod was I have to enter it currently looks like this "automod": {}  lg

### DISCUSSION THREAD

**Discussion by amorking1996** [ScootKit/CustomDCBot#33] [Questions] [answered]
Hello I need help setting up auto mod was I have to enter it currently looks like this "automod": {}

lg

**Comment by SCDerox** [comment, marked as answer]
Hi,
first of all: The moderation module is not longer supported in V2. If you haven't already, please upgrade to V2 as soon as possible to ensure that you are using the newest supported version.
To answer your question: As explained in the [configuration-example-file](https://github.com/SCNetwork/CustomDCBot/blob/a28af29aff2074c47a40d80b7c9270cbdf206a41/modules/moderation/configs/config.json#L112) you can specify what should happen (options: mute, kick, ban) when someone gets x warns. 
Example: Let's say you want to ban everyone when they reach 3 warnings. The configuration should look like this:
```json
"automod": {"3": "ban"}
``

**Comment by amorking1996** [comment]
How is the template if I want to mute someone for a certain amount of time?

**Reply by SCDerox** [reply]
No, mutes are currently permanent

**Reply by SCDerox** [reply]
(at least in your version, at SCNX we have temp-mutes)

**Comment by amorking1996** [comment]
so you can not mute temp with my version

**Reply by SCDerox** [reply]
Please answer to the last thread and don't always create a new one. 
As I said, you can't use temp mutes in your version - which is so outdated (new versions don't include that either, but many other new features).

**Reply by amorking1996** [reply]
where can I find the new versions and what do they contain
